Formula/rkflashtool.rb

```diff
@@ -16,15 +16,9 @@ class Rkflashtool < Formula
   end
 
   depends_on "libusb"
-
-  # Add file 'version.h' that has been forgotten in the tarball
-  resource "version" do
-    url "https://downloads.sourceforge.net/project/rkflashtool/rkflashtool-5.1/version.h"
-    sha256 "b7bd8e3c66170d5f6e6a68c5add677ad365e37b4121251de87ad29e7fdf06cd6"
-  end
+  depends_on "pkg-config" => :build
 
   def install
-    resource("version").stage { buildpath.install "version.h" } if build.stable?
     system "make"
```

Comment:
Thank you for a suggestions, looks like I've sorted this out and this formula is working
As I've deleted forked repo previously, I'll file a new PR